src/main/java/github/scarsz/discordsrv/util/WebhookUtil.java

```diff
@@ -66,14 +68,27 @@ public static void deliverMessage(TextChannel channel, Player player, String mes
         Bukkit.getScheduler().runTaskAsynchronously(DiscordSRV.getPlugin(), () -> {
             try {
                 String avatarUrl = DiscordSRV.config().getString("Experiment_WebhookChatMessageAvatarUrl");
+                String username = DiscordUtil.strip(player.getDisplayName());
+
+                String userId = DiscordSRV.getPlugin().getAccountLinkManager().getDiscordId(player.getUniqueId());
+                if (userId != null) {
+                    Member member = DiscordUtil.getMemberById(userId);
+                    if (member != null) {
+                        if (DiscordSRV.config().getBoolean("Experiment_WebhookChatMessageAvatarFromDiscord"))
+                            avatarUrl = member.getUser().getAvatarUrl();
```

Comment:
```suggestion
                            avatarUrl = member.getUser().getEffectiveAvatarUrl();
```
Effective avatar (default avatar if a avatar isn't set), otherwise good to go

Reply:
Oh this was already merged, let me push this to develop.

Reply:
https://github.com/DiscordSRV/DiscordSRV/commit/485d67107c9b3097f85c34d7f4dd83b6d24025e9